.gitignore

```diff
@@ -1,2 +1,261 @@
 # local VSCode extension setup
-.vscode
\ No newline at end of file
+.vscode
```

Comment:
Most of the changes in this PR are the same as the #14 changes that were already merged.
Have you already merged the latest `master` into the `connect-db` branch?

Reply:
yes, I have- Timea and I were not clear on the merging protocol, and therefore I have this one as well, because I made a small change in the short time. I merged it because if any team members pull from main, they would not be connected to the DB, and wanted to fix that. 
With that, I am currently still exploring the current express app, and making some more changes.. so my next push to my branch will be bigger changes. 

Going back to the merge protocol, 
Should all pull requests be approved by you before they are pushed?

Reply:
> Going back to the merge protocol,
Should all pull requests be approved by you before they are pushed?

Better not, probably you and Timea will get in a work speed and level of detail that I may not keep up with, better not lock the project like that.
If someone has a question about certain feature or implementation of the feature, let's discuss between us. I'm always available to discuss any issue, just lemme know when you want feedback.

Still, we have a lot of code in this PR that was already merged, could you try to create another PR, it can be a draft PR, to check if the file diff shows correctly?

Reply:
Sorry to interrupt, but I think the problem is that after the first merge the main and this branch wasn't synchronized, the main didn't get merged into connect-db, so with the second merge every commit turned up again. 
@Ostend if you merge the main into connect-db and create a new PR, that could solve it I think 😳

Reply:
Thanks for the info @Moomia !
It's all good the PR for me, but better fix this issue with the diff.

Reply:
I will cancel this PR, as I have changed the structure even more.. and add a new PR shortly